src/stylesheets/components/_site-alert.scss

```diff
@@ -0,0 +1,61 @@
+// Alert variables ---------- //
```

Comment:
This component needs some mobile-first applied. Better late than never to be mobile first!

Reply:
@thisisdano There isn't anything I am seeing that is responsive. Is there something that stands out?

Reply:
I'm seeing this on mobile. I'll take a look. <img width="204" alt="Screen Shot 2020-10-30 at 11 29 42 AM" src="https://user-images.githubusercontent.com/1385752/97732598-bf37dc80-1aa4-11eb-934d-ce44d4d28296.png">

Reply:
<img width="501" alt="Screen Shot 2020-11-06 at 9 19 18 AM" src="https://user-images.githubusercontent.com/1385752/98383495-2bb95b00-2012-11eb-8c8c-5d2d065d5b7d.png">
<img width="453" alt="Screen Shot 2020-11-06 at 9 18 48 AM" src="https://user-images.githubusercontent.com/1385752/98383498-2d831e80-2012-11eb-8f5f-80178ae0268d.png">